s.js

```diff
@@ -0,0 +1,16 @@
+const { getPackages } = require('@lerna/project');
```

Comment:
What's this file? Maybe a leftover or ended up in the wrong place?

Reply:
That's cruft, good catch